Shared/af.lproj/AuthenticationManager.strings

```diff
@@ -1,4 +1,4 @@
-/* Text displayed above the input field when changing the existing passcode */
+/* Text dplayed above the input field when changing the existing passcode */
```

Comment:
Change to show that PR does not run the test if a file not related to the app or tests (as decided in a list) is modified
https://app.bitrise.io/app/6c06d3a40422d10f/pipelines/903644ff-c690-48ae-b3fa-37b35de3211e
UI tests do not run based on the ENV VAR